libs/utils/utils.js

```diff
@@ -878,6 +891,9 @@ export async function loadArea(area = document) {
     delayed([getConfig, getMetadata, loadScript, loadStyle]);
   }
 
+  // placing the hash back to scroll the screen.
+  window.location.hash = currentHash;
```

Comment:
nit: only set this if `currentHash` has a value

Reply:
done